store/presenters/presenters.go

```diff
@@ -28,44 +28,51 @@ func LogListeningAddress(address common.Address) string {
 }
 
 // ShowEthBalance returns the current Eth Balance for current Account
-func ShowEthBalance(store *store.Store) (string, error) {
+func ShowEthBalance(store *store.Store) ([]interface{}, error) {
 	if !store.KeyStore.HasAccounts() {
 		logger.Panic("KeyStore must have an account in order to show balance")
 	}
 	account, err := store.KeyStore.GetAccount()
 	if err != nil {
-		return "", err
+		return nil, err
 	}
 	address := account.Address
 	balance, err := store.TxManager.GetEthBalance(address)
 	if err != nil {
-		return "", err
+		return nil, err
 	}
-	result := fmt.Sprintf("ETH Balance for %v: %v", address.Hex(), balance)
 	if balance.IsZero() {
-		return result, errors.New("0 Balance. Chainlink node not fully functional, please deposit ETH into your address: " + address.Hex())
+		return nil, errors.New("0 Balance. Chainlink node not fully functional, please deposit ETH into your address: " + address.Hex())
 	}
-	return result, nil
+	keysAndValues := []interface{}{
```

Comment:
Let's use a `map[string]interface{}` to be the data structure for `keysAndValues`. An array doesn't treat keys with the importance that we would like. Yes, it's ok that this changes the return value type.